src/components/Components/ComponentDetails/tabs/ComponentDetailsTab.tsx

```diff
@@ -60,6 +63,27 @@ const ComponentDetailsTab: React.FC = () => {
         <ComponentBuildSettings component={component} />
         <ComponentNudgesDependencies component={component} />
       </DetailsSection>
+      <DetailsSection
+        title="Registry Login Information"
+        description={
+          <span>
+            Use this information for accessing the registry for build images.
+            {LEARN_MORE_ABOUT_REGISTRY_LOGIN_URL && (
```

Comment:
We shouldn't need condition for constant defined in our own app. you can add the full link once you have the correct docks link until then we can just omit this part